basket/news/tasks.py

```diff
@@ -1350,3 +1346,34 @@ def snitch(start_time=None):
     requests.post('https://nosnch.in/{}'.format(snitch_id), data={
         'm': totalms,
     })
+
+
+def get_fxa_user_data(fxa_id, email):
```

Comment:
Does it make sense to use this function here too?

https://github.com/mozmeao/basket/blob/a1df42d257d0b8bb63c41882eabd05b24adf7fe3/basket/news/tasks.py#L749